src/components/CorporateCampaign/CampaignThanks.vue

```diff
@@ -160,6 +161,14 @@ export default {
 				userHasLentBefore(hasLentBefore);
 				userHasDepositBefore(hasDepositBefore);
 
+				// MARS-246 Hotjar user attributes
+				setHotJarUserAttributes({
+					userId: this.userId,
+					hasEverLoggedIn: true,
```

Comment:
Can this page only be accessed by logged in user?

Reply:
The page is open but this component would only show up after user was logged in and successfully executed a checkout...